apps/src/lib/levelbuilder/lesson-editor/CreateNewLevelInputs.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import Button, {ButtonColor} from '@cdo/apps/templates/Button';
+
+//TODO: Set up creating a new level. Make sure dropdown has all level types
+
+export default class AddLevelFilters extends Component {
```

Comment:
please fix class name

Reply:
Huh, how does the code even work as-is? Is it because of `export default`?

Reply:
exactly. importing the default implicitly sets the name of the import.